packages/components/src/Lettera/Models.purs

```diff
@@ -276,6 +277,7 @@ data BodyElement
   | Footnote String
   | Question String
   | Quote String
+  | Ad String
```

Comment:
Let's add a comment here that this block doesn't come from Lettera

Reply:
Yes, so I pushed this comment. Is it clear enough?

```
  -- Note that Ad does NOT come from Lettera, but was added here to make smart ad placement possible
  | Ad String
```

(So @vapaj did catch this, as you thought he would @ae-mo 😄 )

Reply:
Yes, perfect 😅